src/main/java/org/codeforamerica/messaging/controllers/MessageController.java

```diff
@@ -34,10 +34,9 @@ public ResponseEntity<Message> createMessage(@Valid @RequestBody MessageRequest
 
     }
 
-
     @GetMapping("/{id}")
     public ResponseEntity<Optional<Message>> getMessage(@PathVariable Long id) {
-        Optional<Message> message = messageService.getMessage(id);
+       Optional<Message> message = messageService.getMessage(id);
```

Comment:
Looks like a space was accidentally deleted here

Reply:
No idea how it happened. I can fix it.

![image](https://github.com/codeforamerica/messaging/assets/122047394/4d274976-e2ab-4cdd-824e-12da46c35b6d)